specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/stable/2025-06-01/diagnostic.json

```diff
@@ -0,0 +1,307 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the metadata of Spot Placement Scores.": {
+            "$ref": "./examples/GetSpotPlacementScores.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/placementScores/spot/generate": {
+      "post": {
+        "tags": [
+          "spotPlacementScores"
+        ],
+        "operationId": "SpotPlacementScores_Post",
+        "description": "Generates placement scores for Spot VM skus.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "spotPlacementScoresInput",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SpotPlacementScoresInput"
+            },
+            "description": "SpotPlacementScores object supplied in the body of the Post spot placement scores operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SpotPlacementScoresResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Returns spot VM placement scores for given configurations.": {
+            "$ref": "./examples/GenerateSpotPlacementScores.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ComputeDiagnosticBase": {
+      "description": "Contains metadata of a diagnostic type",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DiagnosticProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DiagnosticProperties": {
+      "description": "Contains additional properties of a diagnostic",
+      "type": "object",
+      "properties": {
+        "supportedResourceTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Describes what are the supported resource types for a diagnostic."
+        }
+      }
+    },
+    "ErrorResponse": {
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=5&length=13)

This definition is unused and should be removed.

Reply:
Removed